modules/tomcat/templates/proxy.conf.epp

```diff
@@ -1,10 +1,10 @@
-<%- | String $host_name
+<%- | String $dns_name
```

Comment:
All  template should be moved to httpd module

Reply:
moved to web profile